test/test_package_loading.py

```diff
@@ -1,13 +1,29 @@
 import pytest
 
 
-def test_load_from_url(selenium_standalone):
+def test_load_from_url(selenium_standalone, web_server_secondary):
```

Comment:
Do we want to keep the old version of this test around as well?  (To test that you can load by URL from the same origin as well -- not that there's a real necessary use case for that, but it should probably be expected to work...)